extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-tag.js

```diff
@@ -75,5 +72,21 @@ describes.realWin(
       expect(shoppingTag.element.textContent).to.be.empty;
       expect(shoppingTag.isLayoutSupported(Layout_Enum.CONTAINER)).to.be.true;
     });
+
+    it('should set active product in store service when shopping tag is clicked', async () => {
```

Comment:
Does this test simulate a click? It looks like `shoppingDataDispatchStoreService` does not.

Reply:
Sounds good, I added the click simulation.

Reply:
I see the click simulation but the click isn't setting the data in this test,`shoppingDataDispatchStoreService` is. 

If we want to test that the click is what's setting the data then we shouldn't need `shoppingDataDispatchStoreService`.

Reply:
Yes, sounds good, removed the call to `shoppingDataDispatchStoreService`